ts/pages/instant/configurator.tsx

```diff
@@ -59,7 +59,7 @@ export class Configurator extends React.Component {
 <html>
     <head>
         <meta charset="utf-8" />
-        <script src="https://instant.0x.org/v3/instant.js"></script>
+        // <script src="https://instant.0x.org/v3/instant.js"></script>
```

Comment:
```suggestion
```

You can just remove it altogether. We would need a different style of comment in JSX.